bfvmm/include/intrinsics/intrinsics_intel_x64.h

```diff
@@ -79,8 +84,18 @@ class intrinsics_intel_x64 : public intrinsics_x64
     virtual bool vmread(uint64_t field, uint64_t *val)
     { return __vmread(field, val); }
 
+    virtual uint64_t vmread(uint64_t field)
```

Comment:
I don't think it really matters once the exception logic is complete. If the read fails, the exception logic will kill your attempt to continue unless you explicitly catch the failure, and manually change the corrupted bit in the VMCS.